docs/styling.md

```diff
@@ -71,63 +72,81 @@ As an example, here is [a page with `fsdocs-navbar-position` set to `fixed-right
+| `.fsdocs-member-tooltip` | tooltips in generated member lists      |
+| `.fsdocs-member-xmldoc`  | documentation in generated member lists |
+| `.fsdocs-entity-list`    | generated entity lists                  |
+| `.fsdocs-entity-name`    | generated entity lists                  |
+| `.fsdocs-entity-xmldoc`  | documentation in generated entity lists |
+| `.fsdocs-exception-list` | generated exception lists               |
+| `.fsdocs-summary`        | the 'summary' section of an XML doc     |
+| `.fsdocs-remarks`        | the 'remarks' section of an XML doc     |
+| `.fsdocs-params`         | the 'parameters' section of an XML doc  |
+| `.fsdocs-param`          | a 'parameter' section of an XML doc     |
+| `.fsdocs-param-name`     | a 'parameter' name of an XML doc        |
+| `.fsdocs-returns`        | the 'returns' section of an XML doc     |
+| `.fsdocs-example`        | the 'example' section of an XML doc     |
+| `.fsdocs-note`           | the 'notes' section of an XML doc       |
+| `.fsdocs-para`           | a paragraph of an XML doc               |
 
 Some generated elements are given specific HTML ids:
 
-|  HTML Element Id    | Content|  
-|:------------------------------------|:--------------------------------------------------------------|
-| `#fsdocs-content`              |    The generated content |  
-| `#fsdocs-searchbox `      |   The search box |
-| `#fsdocs-logo `      |  The logo |
-| `#fsdocs-nav `      |  The navigation-bar |
+| HTML Element Id      | Content               |
+| :------------------- | :-------------------- |
+| `#fsdocs-content`    | The generated content |
+| `#fsdocs-searchbox ` | The search box        |
+| `#fsdocs-logo `      | The logo              |
+| `#fsdocs-nav `       | The navigation-bar    |
 
 If you write a new theme by CSS styling please contribute it back to FSharp.Formatting.
 
 ## Customizing via a new template
 
-You can do advanced styling by creating a new template.  Add a file `docs/_template.html`, likely starting
+You can do advanced styling by creating a new template. Add a file `docs/_template.html`, likely starting
 with the existing default template.
 
 > NOTE: To enable hot reload during development with `fsdocs watch` in a custom `_template.html` file,
-> make sure to add the single line `{{fsdocs-watch-script}}`  to your `<head>` tag.
+> make sure to add the single line `{{fsdocs-watch-script}}` to your `<head>` tag.
 
 > NOTE: There is no guarantee that your template will continue to work with future versions of F# Formatting.
 > If you do develop a good template please consider contributing it back to F# Formatting.
 
+## Customizing menu items by template
+
+You can add advanced stylings to the sidebar generated menu items by creating a new template for it.
+To customize the generated headers, add template file `docs/_menu_template.html`. A starting template with the default configuration would look like:
```

Comment:
Mention they should put a file in the `--input` folder, by default that is typically the `docs` folder but it could be something different.

Both templates should exist in order for this to work, please add a note for this as well.